### DISCUSSION TITLE
Model convergency issue

### DISCUSSION THREAD

**Discussion by hjia1005** [MODFLOW-ORG/modflow6#1254] [Q&A] [answered]
I am running a model calibration using pestpp-ies. With the random realizations of parameters, the steady-state takes a long time to converge. As a solution, I split the original model into a steady-state model and the transient model. I set the OUTER_MAXIMUM as 10000 for the steady-state model. In my workflow, after steady state is done, the result head is read as the strt for the transient model which has a much lower outer_maximum (300) and ATS is also on. But for the convenience of post-processing, I included the steady state as the first stress period in the transient model, and the first stress period normally takes less than 1 second to run as planned. This strategy worked for 99% of the model runs until I saw one failure, see below. I set both OUTER_DVCLOSE and INNER_DVCLOSE as 0.001 for the transient model, but in this model, it refused to converge even with a low value of 5e-6. May I ask how to avoid this and make it converge?

![image](https://github.com/MODFLOW-USGS/modflow6/assets/33449394/9717dab9-b344-40f0-b2fd-69c9351cda7e)

**Comment by hjia1005** [comment]
For most of the other model runs, it's like this:

![image](https://github.com/MODFLOW-USGS/modflow6/assets/33449394/9bbefe51-07cf-4285-9f98-2912e5e123d3)

**Comment by hjia1005** [comment, marked as answer]
It seems this is the solution, to update the NO_PTC in the ims file. I will give it a try.

BEGIN options
     PRINT_OPTION  all
     COMPLEXITY  complex
     **NO_PTC first**
END options

**Reply by RyanConway91** [reply]
I have found PCT often causes problems with complex models with multiple timesteps/stress periods. Turning it off is the first thing I try with with convergence issues. It certainly can help in some cases, but not all.
